_includes/component-pattern-overview.html

```diff
@@ -25,8 +25,8 @@ <h4>{{page.description}}</h4>
             <img src="../assets/img/{{page.pageCategory}}-overview/{{item.title | downcase | replace: ' ', '-'}}.png"
               class="card-img-top" alt="{{item.title | append: ' image' }}">
             <div class="card-body">
-              <a class="stretched-link" href="/{{page.pageCategory}}/{{item.title | replace: ' ', '-' | downcase}}/">
-                <h5 class="card-title">{{item.title}}</h5>
+              <a class="stretched-link link-primary" href="/{{page.pageCategory}}/{{item.title | replace: ' ', '-' | downcase}}/">
+                <p class="h5 text-primary card-title">{{item.title}}</p>
```

Comment:
Try not to stack paragraph tag with a .h5 class. Use a <div> instead
```suggestion
                <div class="h5 text-primary card-title">{{item.title}}</div>
```